bundles/de.uka.ipd.sdq.dsexplore.analysis.simulizar/src/de/uka/ipd/sdq/dsexplore/analysis/simulizar/SimulizarJob.java

```diff
@@ -47,7 +47,10 @@ private static SimuLizarWorkflowConfiguration updateConfig(SimuLizarWorkflowConf
 	@Override
 	protected void addSimulatorSpecificJobs(SimuLizarWorkflowConfiguration configuration) {
 		this.add(new PathChangerJob(configuration));
-		this.add(new PCMInterpreterRootCompositeJob(configuration));
+        this.add(SimuLizarPlatform.getPlatformComponent()
+                .analysisFactory()
+                .create((SimuLizarWorkflowConfiguration) configuration)
+                .rootJob());
```

Comment:
Formatting looks broken here. A quick auto-formatting using the predefined formatter in our coding convetions should do the trick.
https://github.com/PalladioSimulator/Palladio-Build-CodingConventions